typescript/src/events/reactions/rawReactionAddBlockList.ts

```diff
@@ -25,9 +24,10 @@ export class UserModerationEvent extends ModerationEvent<ArgumentType, unknown>
 	};
 
 	public async run(data: LLRCData, emoji: string) {
-		const [enabled, blockedReactions, ignoredChannels, softAction, hardAction, adder] = await data.guild.readSettings((settings) => [
+		const [enabled, blockedReactions, channelId, ignoredChannels, softAction, hardAction, adder] = await data.guild.readSettings((settings) => [
```

Comment:
```suggestion
		const [enabled, blockedReactions, logChannelId, ignoredChannels, softAction, hardAction, adder] = await data.guild.readSettings((settings) => [
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.